app/api/create-agent/route.ts

```diff
@@ -0,0 +1,47 @@
+import { NextResponse } from "next/server";
+import { z, ZodError } from "zod";
+import { cookies } from "next/headers";
+import { spawn } from "child_process";
+
+const createAgentSchema = z.object({
+  name: z.string().min(1),
```

Comment:
There should be no `name` parameter. There should already be a constant somewhere representing the production agent